app/models/user.rb

```
@@ -1121,20 +1121,43 @@ def should_subscribe_preview_be_regenerated?
+                  saved_change_to_bio?
+
+    Iffy::Profile::IngestJob.perform_async(id)
+  end
+
+  def has_completed_payouts?
+    payments.completed.exists? ||
+      made_a_successful_sale_with_a_stripe_connect_or_paypal_connect_account?
+  end
+
+  # Add this to group it with other associations (usually at the top of the file)
+  has_many :sales, through: :products
```

RuboCop (1.75.5)

[convention] Inconsistent indentation detected.

(Layout/IndentationConsistency)